PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchView.swift

```diff
@@ -103,18 +106,48 @@ struct SearchEmptyView: View {
     }
 
     var body: some View {
-        if let text = viewModel.buttonText {
-            EmptyStateView(viewModel: viewModel) {
-                GetPocketPremiumButton(text: text)
+        if let buttonType = viewModel.buttonType {
```

Comment:
main reason why I decided to create a Button Type is because we were relying on `buttonText` to create a Premium button. However, now we have to consider another button type. Future cases may be needed, but added `normal` for now

Reply:
So the naming seems uneven;
.premium is for upselling to premium
.normal is for reporting an issue

It seems like there is a 1-1 connection between the button type and it's purpose, as in I do see a way we could use .normal for anything other than reporting an issue here. So why don't we name it .reportIssue instead?

Reply:
I hadn't seen the emptyView case...
Drop this to a nitpick then, and go with your gut.

Reply:
I incorporated this nitpick, since no preference either way. Let me know if you prefer it!